src/Assets/Pack/build/Microsoft.AspNetCore.Assets.Pack.targets

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <Import Project="$(MSBuildThisFileDirectory)Microsoft.AspNetCore.StaticWebAssets.targets" />
+  <Import Project="$(MSBuildThisFileDirectory)Microsoft.AspNetCore.StaticWebAssets.endpoints.targets" />
+
+  <Import Project="$(MSBuildThisFileDirectory)Microsoft.AspNetCore.StaticWebAssets.debug.targets"
+    Condition="'$(Configuration)' == 'Debug'" />
```

Comment:
Do not tie the import directly to the `Debug` configuration. 

Create a prop instead `UseDebugBlazorAssets` or similar and provide a default based on the config if you choose to do so. That allows people to unconditionally enable/disable it.